EdFi.Tools.ApiPublisher.Cli/plainTextNamedConnections.json

```diff
@@ -1,16 +1,16 @@
 {
   "connections": [
     {
-      "name": "devSource",
+      "name": "Local_v5.2",
       "url": "http://localhost:54746/",
-      "key": "RDnTsr7bO8Wm6FiyhBMup",
-      "secret": "vEJ4iaMLfVp6FdQ7RTP9U"
+      "key": "LOCAL-SANDBOX-KEY",
+      "secret": "LOCAL-SANDBOX-SECRET"
     },
     {
-      "name": "devTarget",
-      "url": "http://localhost:54746/",
-      "key": "fK2aPG5ihEprXBDcj0WtL",
-      "secret": "IlBjywACL2r1YVHdZTDaO"
+      "name": "Hosted_Sample_v5.2",
+      "url": "https://api.ed-fi.org/v5.2/api/",
+      "key": "RvcohKz9zHI4",
+      "secret": "E1iEFusaNf81xzCxwHfbolkC"
```

Comment:
I feel a bit uncomfortable about this. At the same time, any determined person can easily pull this key and secret from the SwaggerUI instance.